scripts/contrib/pad_model_vocabulary.py

```diff
@@ -0,0 +1,52 @@
+#!/usr/bin/env python3
+# Pads a Marian model's vocabulary to have greater size.  The added tokens have
+# zero probability.
+# ./pad_model_vocabulary.py input.npz output.npz desired_vocab_size
+#
+# You'll also need to separately pad your vocabulary file like so:
+# old=$(wc -l input.vocab |cut -d " " -f 1)
+# (cat input.vocab; seq -f "<PADDING%g>" $((desired_vocab_size-old))) >output.vocab
```

Comment:
This script isn't actually needed for self-adaptive Marian. It came with the swappable code and found it's way into here because i mistakenly thought it was part of Marian. I've removed it.